### PR TITLE
Add OSP13 support for ceph checks

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by hmakkapati** [comment]
recheck_trusty_mitaka_deploy

**Comment by tonytan4ever** [comment]
recheck_trusty_mitaka_deploy
recheck_trusty_newton_deploy
recheck_xenial_pike_deploy

File: playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2 [hunk 1]
```diff
@@ -2,8 +2,11 @@
 {% set label = "ceph_cluster_stats" %}
 {% set check_name = label+'--'+inventory_hostname %}
 {% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
-{% if maas_rpc_legacy_ceph | bool %}
-{%   set _ = ceph_args.extend(["--container-name", container_name]) %}
+{% if (maas_rpc_legacy_ceph | bool) or deploy_osp %}
```
Comment (hmakkapati) [inline playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: Do we need `deploy_osp|default(False)` here?

File: playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2 [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
+{% if deploy_osp %}
```
Comment (hmakkapati) [inline playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: Same

File: playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2 [hunk 3]
```diff
@@ -2,8 +2,11 @@
 {% set label = "ceph_mon_stats" %}
 {% set check_name = label+'--'+inventory_hostname %}
 {% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
-{% if maas_rpc_legacy_ceph | bool %}
-{%   set _ = ceph_args.extend(["--container-name", container_name]) %}
+{% if (maas_rpc_legacy_ceph | bool) or deploy_osp %}
+{% if deploy_osp %}
```
Comment (hmakkapati) [inline playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2]: Same here and the next line

File: playbooks/templates/rax-maas/ceph_osd_stats.yaml.j2 [hunk 4]
```diff
@@ -2,8 +2,11 @@
 {% set label = "ceph_osd_stats" %}
 {% set check_name = label+'--'+inventory_hostname %}
 {% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
-{% if physical_host is defined and physical_host != ansible_host %}
-{%   set _ = ceph_args.extend(["--container-name", container_name]) %}
+{% if (physical_host is defined and physical_host != ansible_host) or deploy_osp %}
```
Comment (hmakkapati) [inline playbooks/templates/rax-maas/ceph_osd_stats.yaml.j2]: Same here and the next line

File: playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2 (hunk 1 above)
Comment (tonytan4ever) [reply playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: Good catch,  done.

File: playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2 (hunk 2 above)
Comment (tonytan4ever) [reply playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: Done

File: playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2 (hunk 3 above)
Comment (tonytan4ever) [reply playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2]: Done

File: playbooks/templates/rax-maas/ceph_osd_stats.yaml.j2 (hunk 4 above)
Comment (tonytan4ever) [reply playbooks/templates/rax-maas/ceph_osd_stats.yaml.j2]: Done.

File: playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2 [hunk 5]
```diff
@@ -2,8 +2,11 @@
 {% set label = "ceph_mon_stats" %}
 {% set check_name = label+'--'+inventory_hostname %}
 {% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
-{% if maas_rpc_legacy_ceph | bool %}
-{%   set _ = ceph_args.extend(["--container-name", container_name]) %}
+{% if (maas_rpc_legacy_ceph | bool) or deploy_osp %}
```
Comment (hmakkapati) [inline playbooks/templates/rax-maas/ceph_mon_stats.yaml.j2]: Missing default here

**Pull request merged**